src/components/views/spaces/SpacePanel.tsx

```diff
@@ -221,14 +221,20 @@ const SpacePanel = () => {
                             space={s}
                             activeSpaces={activeSpaces}
                             isPanelCollapsed={isPanelCollapsed}
-                            onExpand={() => setPanelCollapsed(false)}
+                            onExpand={() => {
+                                closeMenu();
```

Comment:
That looks like a good candidate to go in a `useEffect` definition rather than explicitely calling it for every `setPanelCollapsed(false)` call